charts/jaeger/values.yaml

```
@@ -196,6 +196,10 @@ agent:
   extraEnv: []
   daemonset:
     useHostPort: false
+    updateStrategy:
+      type: RollingUpdate
+      rollingUpdate:
+        maxUnavailable: 1
```

```suggestion
    updateStrategy: {}
      # type: RollingUpdate
      # rollingUpdate:
      #   maxUnavailable: 1
```

@mehta-ankit I just merged, would you mind following up with a PR that sets this to empty so that default is implicitly set?

---

https://github.com/jaegertracing/helm-charts/pull/184